features/exploredash/src/main/java/org/dash/wallet/features/exploredash/repository/ExploreRepository.kt

```diff
@@ -68,8 +67,6 @@ class GCExploreDatabase @Inject constructor(
         private val log = LoggerFactory.getLogger(GCExploreDatabase::class.java)
     }
 
-    private var contextRef: WeakReference<Context> = WeakReference(context)
```

Comment:
This is getting forcefully unpacked anyway. Might as well use the `context` directly.